source/indexes.txt

```diff
@@ -74,6 +74,17 @@ For each index, |compass-short| displays the following information:
      - Any special properties (such as uniqueness, partial) of the
        index.
 
+Required Roles 
+--------------
+
+To :ref:`create <compass-create-index>`,
+:ref:`drop <compass-drop-an-index>`, or :ref:`hide <compass-hide-an-index>` 
+indexes, you must have access provided by at least one of the following roles:
+
+- :authrole:`Project Owner` or :authrole:`Organization Owner`
+- :authrole:`Project Data Access Admin`
```

Comment:
It's possible to view indexes with the `Project Data Access Read/Write`, `Project Data Access Read Only`, and `Project Data Access Admin` roles as well. Is that worth calling out separately?